iree/tools/iree-run-module-main.cc

```diff
@@ -160,6 +160,15 @@ iree_status_t Run() {
 
 extern "C" int main(int argc, char** argv) {
   iree_flags_parse_checked(IREE_FLAGS_PARSE_MODE_DEFAULT, &argc, &argv);
+  if (argc > 1) {
+    // Avoid iree-run-module spinning endlessly on stdin if the user uses single
+    // dashes for flags.
+    std::cout << "Error: More than one positional argument (single dash '-') "
+                 "given for initiating stdin input. Perhaps you meant to use "
+                 "'--' instead?"
+              << std::endl;
+    return 3;  // status IREE_STATUS_INVALID_ARGUMENT
```

Comment:
Seems weird to reference IREE_STATUS_* here. If we want program return codes to match IREE statuses (not sure if that's a think we do elsewhere?) then we should use it directly, but I suspect we don't want that as the precise enum value of a status code seems like an implementation detail. I would either return `1` here or you can abort, as that's what the other failures here do